docs/quick-start.md

```diff
@@ -4,7 +4,7 @@ sidebar_position: 10
 description: Quick start for running Inspektor Gadget.
 ---
 
-The following examples use the [trace_open](./gadgets/trace_open) Gadget which triggers when a file is open on the system.
+The following examples use the [trace_open](../gadgets/trace_open/README.md) Gadget which triggers when a file is open on the system.
```

Comment:
Hmm. I don't relative links will work for website. At least it fails for my local rendered version.

Also: https://github.com/inspektor-gadget/inspektor-gadget/pull/2247#discussion_r1407985276

Reply:
Thanks! Fixed it on my last push!